change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -1164,17 +1166,17 @@ PROCEDURE CLEANUPTEXT
 END [PROCEDURE]
 
 PROCEDURE VALIDATERGDATA
-[* Check the member's new street address to see if it contains
-** a PO Box address per the CU's configuration settings. If a
+[* Check the member's new street and new extra address to see if it
+** contains a PO Box address per the CU's configuration settings.
 ** PO Box address is found, set an error code and create a
-** note record on the member's account. Also validates NEWSTREET,
-** NEWCITY, NEWSTATE and NEWZIPCODE are completed.
+** If a note record on the member's account. Also validates
+** NEWSTREET, NEWCITY, NEWSTATE and NEWZIPCODE are completed.
 ** PASS    NEWSTREET
 ** PASS    NEWCITY
 ** PASS    NEWSTATE
 ** PASS    NEWZIPCODE
-** PASS    PARAMPORESTRICT Boolean flag: should the street be
-**         checked for a PO Box address
+** PASS    PARAMPORESTRICT Boolean flag: should the street and extra
```

Comment:
a word or words at the end of the sentence??

Reply:
Good find @tkainz ! I've made the change in a new commit. Can you re-review?